src/wm_mecanum_teleop_node.py

```diff
@@ -49,7 +49,7 @@ def callback(self, joy):
         # x axis linear velocity
         twist.linear.x = self.maxLinearVelocity * vLinear * cos(Heading)
         # y axis linear velocity
-        twist.linear.y = self.maxLinearVelocity * vLinear * sin(Heading)
+        twist.linear.y = self.maxLinearVelocity * vLinear * sin(Heading) * -1
```

Comment:
Sure about that? It worked well before. If there is somewhere to change it is probably in the teleop.